lighthouse-core/config/default-config.js

```diff
@@ -151,6 +151,7 @@ const defaultConfig = {
       'seo/robots-txt',
       'seo/tap-targets',
       'accessibility',
+      'issues',
```

Comment:
how would you feel about something slightly more specific like `audits-issues` or something more closely tying it to audits protocol?

Reply:
Having issues live in the Audits namespace in CDP makes coming up with a nice name tricky. I was thinking audits-issues as well but audits in this case doesn't make sense in a LH context. I was also thinking of maybe `inspector-issues` as that's the top-level type for all issues.

Reply:
I could get behind `inspector-issues` 👍 

agreed `audits-issues` is a bit weird

Reply:
`inspector-issues` SGTM too

(though is `inspector` too ancient history for most folks these days? something like `devtools-issues` is basically synonymous and may be more obvious what it's referring to)

Reply:
>  I was also thinking of maybe `inspector-issues` as that's the top-level type for all issues.

actually you already pointed out that's exactly what they're called in the protocol so ignore my earlier parenthetical :)

`inspector-issues` SGTM